src/FakeItEasy/Configuration/ArgumentCollection.cs

```diff
@@ -11,7 +11,9 @@ namespace FakeItEasy.Configuration
     /// <summary>
     ///   A collection of method arguments.
     /// </summary>
+#if FEATURE_BINARY_SERIALIZATION
     [Serializable]
+#endif

```

Comment:
For my understanding, this means that users of FIE on .NET Core would not be able to take advantage of the serializability of classes? Not that I'm sure what that was ever used for, except for the recorders. Which are probably barely used either…